css/style.css

```diff
@@ -111,6 +112,12 @@ video {
 .pointer-list__oval--black{
```

Comment:
Add a space between the class name and {

Reply:
in the next commit i will do that.